guardian/cdk-cli
# Tidy up the CLI in preparation for migration fortnight

## What does this change?

This PR removes the `migrate` and `init` commands.

It also changes the `new` command to create the project boilerplate by default as we consider this to be the most frequently desired use-case within the department.

Finally, some old sections of the README have been removed as they are no longer relevant.

## How to test

I've tested this by running `./bin/run new --output=../test-project --app=Amigo --stack=deploy` and confirming that the generated `cdk` project works as expected.

## How can we measure success?

* There is less potential for confusion as old documentation and commands have been removed
* Named arguments also makes command usage clearer

## Have we considered potential risks?

I don't think there are any significant risks involved in this PR.
